app/helpers/authentication_helper.rb

```diff
@@ -23,6 +23,16 @@ def authentication_enabled_providers_for_user(user = current_user)
     Authentication::Providers.enabled_for_user(user)
   end
 
+  def signed_up_with(user = current_user)
+    providers = Authentication::Providers.enabled_for_user(user)
+
+    # If the user did not authenticate with any provider, they signed up with an email.
+    auth_method = providers.presence ? providers.map(&:official_name).to_sentence : "Email & Password"
+    verb = providers.size > 1 ? "any of those" : "that"
+
+    "Reminder: you used #{auth_method} to authenticate your account, so please use #{verb} to sign in if prompted."
```

Comment:
I don't know why we used to render `Reminder: You used`, but I took the liberty of lowercasing it because...grammar.

Reply:
IIRC we use this because our sessions don't really expire and people tended to forget how they signed up.

Reply:
Heh, I should have worded this more clearly—I was just confused about why we capitalized "You" here 😂 the rationale for the reminder itself makes sense to me though 👍🏽

Reply:
Oh, sorry, I misunderstood 🤦‍♂️

Reply:
It's all good, @citizen428! 🤗 ❤️ My comment wasn't totally clear to begin with :)